### ISSUE TITLE
wundergraph/sdk peer resolution

### ISSUE THREAD

**Issue by scriptjs** [wundergraph/wundergraph#695]
### Bug description

Installing @wundergraph/sdk@npm:0.137.0 results in an inconsistent state with packages is depends on.

```
$ yarn install                         
➤ YN0000: ┌ Resolution step
➤ YN0060: │ @wundergraph/sdk@npm:0.137.0 [9c4f1] provides graphql (p91634) with version 16.6.0, which doesn't satisfy what openapi-to-graphql and some of its descendants request
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 0s 524ms
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed in 3s 647ms
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 1s 899ms
➤ YN0000: Done with warnings in 6s 559ms
```

```
$ yarn explain peer-requirements p91634
➤ YN0000: @wundergraph/sdk@npm:0.137.0 [9c4f1] provides graphql@npm:16.6.0 with version 16.6.0, which doesn't satisfy the following requirements:

➤ YN0000: graphql-scalars@npm:1.20.1 [ca256]      → ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 ✓
➤ YN0000: graphql-subscriptions@npm:1.2.1 [ca256] → ^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0                                ✘
➤ YN0000: graphql-upload@npm:13.0.0 [ca256]       → 0.13.1 - 16                                                                                   ✓
➤ YN0000: openapi-to-graphql@npm:2.6.3 [8ac34]    → ^14.0.0 || ^15.0.0                                                                            ✘

➤ YN0000: Note: these requirements start with openapi-to-graphql@npm:2.6.3 [8ac34]
```

### How to reproduce

1. Include @wundergraph/sdk in project.
2. Install using yarn 3

### Expected behavior

Dependency is fetched with a clean resolution to packages dependent on graphql 16.

### WunderGraph information

No applicable

### Environment & setup

Not applicable

### WunderCtl Version

Not applicable

**Comment by scriptjs** [comment]
This pull request will resolve part of this https://github.com/IBM/openapi-to-graphql/pull/471

**Comment by fiam** [comment]
Hi @scriptjs. Thanks for letting us know. This issue has been fixed in 0.138.
